docs/assistant/security-assistant.asciidoc

```diff
@@ -177,4 +177,17 @@ To enable the knowledge base:
 . Turn on the *Knowledge Base* option.
 . Click *Save*. The knowledge base is now active.
 +
-When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
\ No newline at end of file
+When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
+
+[discrete]
+[[ai-assistant-queries]]
+### Get the most from your queries
+
+Elastic AI Assistant helps you take full advantage of the {elastic-sec} platform to improve your security operations. Its ability to assist you depends on the specificity and detail of your questions. The more context and detail you provide, the more tailored and useful its responses will be. 
+
+To maximize its usefulness, consider using more detailed prompts or asking for additional information. For instance, after asking for an {esql} query example, you could ask a follow-up question like, “Could you give me some other examples?”. You can also ask for clarification or further exposition, for example "Please provide comments explaining the query you just gave".
```

Comment:
Ending punctuation needs to go inside quotes, and a question mark is the only end punctuation needed:
```suggestion
To maximize its usefulness, consider using more detailed prompts or asking for additional information. For instance, after asking for an {esql} query example, you could ask a follow-up question like, “Could you give me some other examples?” You can also ask for clarification or further exposition, for example "Please provide comments explaining the query you just gave."
```